test/sql/timestamp.sql

```diff
@@ -400,22 +400,64 @@ FROM unnest(ARRAY[
     ]) AS time;
 
 SELECT time, time_bucket(INTERVAL '4 day', time::date)
-FROM unnest(ARRAY[
-    date '2017-11-02',
-    date '2017-11-03',
-    date '2017-11-06',
-    date '2017-11-07'
-    ]) AS time;
-
-SELECT time, time_bucket(INTERVAL '4 day', time::date, INTERVAL '2 day')
 FROM unnest(ARRAY[
     date '2017-11-04',
     date '2017-11-05',
     date '2017-11-08',
     date '2017-11-09'
     ]) AS time;
 
+SELECT time, time_bucket(INTERVAL '4 day', time::date, INTERVAL '2 day')
+FROM unnest(ARRAY[
+    date '2017-11-06',
+    date '2017-11-07',
+    date '2017-11-10',
+    date '2017-11-11'
+    ]) AS time;
+
+-- 2019-09-24 is a Monday.
```

Comment:
Maybe a slightly more elaborate comment here on the importance of it being a Monday?

`2019-09-24 is a Monday, and we want to ensure that time_bucket returns the week starting with a Monday as date_trunc does, rather than a Saturday which is the date of the PostgreSQL epoch`